### DISCUSSION TITLE
First screen after login

### DISCUSSION THREAD

**Discussion by benni3105** [Budibase/budibase#6576] [General]
Hi guys,

i am trying to build an app with this amazing platform but i have two questions. Is it possible to redirect a user to a specific app after login instead of showing the app selection page (https://<YOUR APP>.budibase.app/builder/apps). If not, is it possible to change the text "Welcome to the <YOUR ORG> portal. Below you'll find the list of apps that you have access to."?

Thank you for your Help!

**Comment by gerhardmpl** [comment]
@benni3105 I do not think the first is possible yet, but see #2301 for a (bit complicated) workaround.
